src/app/amopis/_components/SideBar/SideBar.tsx

```diff
@@ -1,42 +1,42 @@
 import { Column, Paragraph } from '@amsterdam/design-system-react'
 import {
-  DocumentTextIcon,
-  ExternalLinkIcon,
-  HousingIcon,
-  StopIcon,
-  SuitcaseIcon,
+  CogwheelIcon,
+  DocumentsIcon,
+  FolderIcon,
+  HouseIcon,
+  LineChartUpIcon,
 } from '@amsterdam/design-system-react-icons'
 
 export function Sidebar() {
   return (
     <div className="amopis-sidebar">
       <Column gap="x-large">
         <Column alignHorizontal="center" gap="x-small">
-          <HousingIcon fill="white" width={24} />
+          <HouseIcon fill="white" width={24} />
```

Comment:
Not sure why I didn’t use the `Icon` component here in the past, but two problems arise if I try it now:
1. The maximum `size` of heading-3 is too small to implement the design
2. Horizontal centring in a Column is impossible due to `align-self: baseline`